scripts/install_ceres.sh

```diff
@@ -17,9 +17,12 @@
 set -o errexit
 set -o verbose
 
+VERSION="1.13.0"
+
 # Build and install Ceres.
 git clone https://ceres-solver.googlesource.com/ceres-solver
 cd ceres-solver
+git checkout tags/${VERSION}
```

Comment:
Clone directly the tag to save time if they are huge differences from master and tag?